### PR TITLE
Add datasheet to /openstack page

### LINKED ISSUE #8161: [COPY UPDATE] index.html - "Canonical OpenStack on Ubuntu - the open source private cloud reference architecture"
Please update index.html - "Canonical OpenStack on Ubuntu - the open source private cloud reference architecture" from the [copy doc](https://docs.google.com/document/d/1eqHK0hc9v7Mn-XV36QRyNkJenTXgBQR_-RULKm5rZ5o/edit?usp=drivesdk)

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Add datasheet to /openstack page
- Ant can you re-approve, I had to redo it.

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/openstack
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to the [copy doc](https://docs.google.com/document/d/1eqHK0hc9v7Mn-XV36QRyNkJenTXgBQR_-RULKm5rZ5o/edit#) and resolve comments

## Issue / Card

Fixes #8161

## Screenshots

image

### CODE REVIEW

File: templates/shared/pricing/_openstack-packages.html [hunk 1]
```diff
@@ -60,4 +61,16 @@ <h2 class="p-heading--four">Fully managed cloud</h2>
...
   </div>
   {% endif %}
+  <div class="u-no-fixed-width">
+    <p id="os-footnote">
+      <small>
+      <sup><a href="#os-footnote">*</a></sup> Additional features, functionality and integrations are available via add-ons
+      </small>
+    </p>
+      <p class="p-button">
+        <a href="https://assets.ubuntu.com/v1/d4766546-Private-Cloud-Build_Datasheet.pdf">
```
Comment (anthonydillon) [inline templates/shared/pricing/_openstack-packages.html:71]:
```suggestion
        <a href="https://assets.ubuntu.com/v1/d4766546-Private-Cloud-Build_Datasheet.pdf" class="p-link--external">
```

File: templates/shared/pricing/_openstack-packages.html [hunk 2]
```diff
@@ -60,4 +61,16 @@ <h2 class="p-heading--four">Fully managed cloud</h2>
     <p><a href="/openstack/managed">Fully Managed OpenStack&nbsp;&rsaquo;</a></p>
   </div>
   {% endif %}
+  <div class="u-no-fixed-width">
+    <p id="os-footnote">
+      <small>
+      <sup><a href="#os-footnote">*</a></sup> Additional features, functionality and integrations are available via add-ons
```
Comment (anthonydillon) [inline templates/shared/pricing/_openstack-packages.html:67]:
```suggestion
      <sup>*</sup> Additional features, functionality and integrations are available via add-ons
```

**Review by anthonydillon — APPROVED** [review]
A few tiny changes but LGTM :+1:

File: templates/shared/pricing/_openstack-packages.html (hunk 1 above)
Comment (pmahnke) [reply templates/shared/pricing/_openstack-packages.html:71]: We don't do this on assets.

Comment (anthonydillon) [reply templates/shared/pricing/_openstack-packages.html:71]: I would have thought it was important to show user they are about to leave the site and be shown a PDF?

**Pull request merged**